pipeline/transformer/transformer_sierra/src/main/scala/weco/pipeline/transformer/sierra/transformers/SierraGenres.scala

```diff
@@ -66,7 +68,12 @@ object SierraGenres
     primarySubfields: List[Subfield],
     varField: VarField): List[Concept[IdState.Unminted]] =
     primarySubfields.map { subfield =>
-      val concept = Concept.normalised(label = subfield.content)
-      concept.copy(id = identifyConcept(concept, varField))
+      Concept(
+        id = identifyConcept(
+          ontologyType = "Concept",
+          varField = varField
+        ),
+        label = subfield.content
+      ).normalised
```

Comment:
This illustrates one benefit of moving these to methods on an existing instance of Concept/Period/et al – they'll drop in just fine for identified concepts when that work comes along.